cli/format.go

```diff
@@ -192,6 +195,22 @@ func updateCache(ctx context.Context) func() error {
 					// channel has been closed, no further files to process
 					break LOOP
 				}
+
+				if Cli.Stdin {
+					// dump file into stdout
+					f, err := os.Open(file.Path)
+					if err != nil {
+						return fmt.Errorf("failed to open %s: %w", file.Path, err)
+					} else if _, err = io.Copy(os.Stdout, f); err != nil {
+						return fmt.Errorf("failed to copy %s to stdout: %w", file.Path, err)
+					} else if err = os.Remove(file.Path); err != nil {
+						return fmt.Errorf("failed to remove temp file %s: %w", file.Path, err)
+					}
```

Comment:
Same here as the comment below (because I review from bottom to up for some reason).